src/Tests/dotnet-test.Tests/GivenDotnetTestContainsEnvironmentVariables.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using System.IO;
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Xml;
+using System.Xml.Linq;
+using System.Reflection;
+using dotnet.Tests;
+using System.Runtime.InteropServices;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.Utilities;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Cli.Test.Tests
+{
+    public class GivenDotnetTestContainsEnvironmentVariables : SdkTest
+    {
+        private const string TestAppName = "VSTestEnvironmentVariables";
+        private const string EnvironmentOption = "--environment";
```

Comment:
@KathleenDollard need your opinion on a parameter for passing EnvironmentVariables. is "--environment" too ambiguous?

Reply:
So just to clarify... That option only applies to `dotnet test` command, and is passed-thru to `vstest.console.exe` which uses the same parameter.